jina/parsers/hubble/pull.py

```diff
@@ -0,0 +1,25 @@
+"""Argparser module for hub push"""
+from ..helper import add_arg_group
+
+
+def mixin_hub_pull_parser(parser):
+    """Add the arguments for hub pull to the parser
+    :param parser: the parser configure
+    """
+    gp = add_arg_group(parser, title='Pull')
+    gp.add_argument(
+        'id',
+        type=str,
+        help='The UUID8 of the executor to download',
+    )
+    gp.add_argument(
+        '--docker',
+        action='store_true',
+        default=False,
+        help='If set, pull the Docker image from the Jina Hub registry',
+    )
```

Comment:
isn't `id` scheme already determined if it is docker or not?
`id="jinahub+docker://..."` -> `--docker`
`id="jinahub://..."` -> no `--docker`

then `--docker` is redundant

Reply:
Now, `id` in the argument refers to UUID8 only. `jina hub pull {UUID8}` will pull the uploaded executor package only.

So, here, the argument `id` should be used as the full uri path of the executor?

Reply:
> jina hub pull {UUID8} will pull the uploaded executor package only.

1. isn't this then `jinahub://{UUID}`
2. how is it differed from `.add(uses="jinahub://uri")`?

Reply:
I submitted a new commit. Now, we can directly pass URI of executor, `jina hub pull jinahub(+docker)://{UUID8}`